my_game.py

```diff
@@ -175,7 +175,9 @@ def __int__(self, shot_list, **kwargs):
 
         kwargs['filename'] = "images/ufoBlue.png"
 
-        kwargs['scale'] = SPRITE_SCALING
+        scale_change = random.randint(25, 50)
+
+        kwargs['scale'] = SPRITE_SCALING + (scale_change // 100)
 
```

Comment:
There should only be two sizes of UFOs:
https://github.com/tobiasbp/python-arcade-tuesday-02/issues/31

Let's say big is _1.0_ and small is _0.5_. Multiply the one to use (chosen randomly) with SPRITE_SCALING for the final value for _scale_.